numerous/engine/model/lowering/equations_generator.py

```diff
@@ -117,26 +116,29 @@ def _parse_variables(self):
             full_tag = d_u(sv.id)
             self._parse_variable(full_tag, sv, sv_id)
 
-    def get_external_function_name(self,ext_func):
+    def get_external_function_name(self, ext_func):
         if self.llvm:
             return self._llvm_func_name(ext_func)
         return ext_func
+
     def _llvm_func_name(self, ext_func):
         return ext_func + '_llvm1.<locals>.' + ext_func + '_llvm'
 
     def _parse_equations(self, equations):
         logging.info('make equations for compilation')
         for eq_key, eq in equations.items():
-            vardef = Vardef(llvm = self.llvm)
+            vardef = Vardef(llvm=self.llvm)
 
             func, args, target_ids = function_from_graph_generic(eq[2], eq_key, var_def_=vardef)
 
             eq[2].lower_graph = None
             if self.llvm:
-                func_llvm, signature, args, target_ids = compiled_function_from_graph_generic_llvm(eq[2],
-                                                                                                   eq_key,
-                                                                                                   var_def_=Vardef(llvm=self.llvm),
-                                                                                                   compiled_function=True)
+                func_llvm, signature, args, target_ids = \
+                    compiled_function_from_graph_generic_llvm(eq[2],
+                                                              eq_key,
+                                                              imports=self.imports,
+                                                              var_def_=Vardef(llvm=self.llvm),
+                                                              compiled_function=True)
```

Comment:
I can suggest another style for multiline arguments. It saves vertical space and uses more horizontal space:
```
func_llvm, signature, args, target_ids = compiled_function_from_graph_generic_llvm(
    eq[2], eq_key, imports=self.imports, var_def_=Vardef(llvm=self.llvm), compiled_function=True
)
```